packages/docs/docs/graphql/basic-queries.mdx

```diff
@@ -296,6 +296,22 @@ This query searches for a list of `Patients` named `"Eve"`, living in `"Philadel
   </MedplumCodeBlock>
 </details>
 
+## Query Limits
+
+Because complex GraphQL queries may require many database operations, the cost of the query is estimated before execution,
+and may be rejected by the server if the query will be too expensive to perform all at once. The estimated cost of a
+GraphQL query is calculated based on the number of search operations performed and reference links traversed, using the
+following general formula:
+
+- Each reference link (i.e. `Reference.resource` field) in the query costs _D_ units, where D is the nesting depth of the
+  reference field in the overall query
+- Each search operation (e.g. `*List` fields that take a `_count` argument) costs `C^(1.5 * D)` units, where C is a small
+  number (between 1 and 10) that grows with the requested `_count`, and D is the nesting depth of the search field in the
+  overall query
+
+These costs are added together, and compared to a configurable server limit in order to determine if the query should be
+executed against the database.
```

Comment:
One exampel with a 2-3 level deep query will help solidify this. Can you add a code block inside a <details></details> tag, with the total score up top, and comments for how the depth adds to the complexity?